CHANGELOG.md

```diff
@@ -9,6 +9,37 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.7.0 - 2020-05-29
+
+### Added
+
+- Support for worker Nodes running Windows Server 2019 or higher. [Alpha]
+   * Refer to [Antrea Windows documentation] for usage
+   * A known limitation is that K8s NetworkPolicies are not enforced correctly for Service traffic, due to our reliance on userspace kube-proxy; this will be addressed in a future release
+- Support server certificate verification for Controller APIs; users can provide their own certificates (TLS certificate and corresponding CA certificate) or let the Controller generate them.
+- Add "supportbundles" and "supportbundles/download" API resources to Controller and Agent to retrieve all the relevant information useful for providing support for Antrea, along with the corresponding "antctl supportbundle" command.
```

Comment:
Understood you want to be accurate, but do you think better to introduce the feature in easier to understand words first, like: Add antctl command, controller and agent API to collect support bundles of Antrea  system..., or Add support for collecting Antrea support bundle....

Reply:
You're right. I rephrased accordingly.